lib/puppet/provider/iis_httperror/iis_httperror.rb

```diff
@@ -0,0 +1,177 @@
+require 'json'
+
+Puppet::Type.type(:iis_httperror).provide(:iis_httperror) do
+	confine :operatingsystem => :windows
+	defaultfor :operatingsystem => :windows
```

Comment:
Nit: Your indents seem to be 4 spaces (or tabs) instead of 2. We'll have to change that